extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -16,7 +16,14 @@ public function getSiteJsScriptsHash() {
 			$maxTimestamp = max( $maxTimestamp, $revision['touched'] );
 		}
 
-		return $maxTimestamp;
+		if ( empty( $maxTimestamp ) ) {
+			return 0;
+		}
+
+		$datetime = new \DateTime( $maxTimestamp );
+		$timestamp = $datetime->getTimestamp();
```

Comment:
I think we should use `$datetime->format( 'Y-m-d H:i:s' )` so that the returned format is always the same as the one in the database under the `touched` field.